server/pkg/serv/serv.go

```diff
@@ -16,8 +16,51 @@ type Server struct {
 
 var _ pr12er.Pr12ErServiceServer = (*Server)(nil)
 
-func (s Server) GetDetails(_ context.Context, _ *pr12er.GetDetailsRequest) (*pr12er.GetDetailsResponse, error) {
-	return nil, status.Errorf(codes.Unimplemented, "method GetDetails not implemented")
+func (s Server) GetDetails(_ context.Context, in *pr12er.GetDetailsRequest) (*pr12er.GetDetailsResponse, error) {
+	resp := &pr12er.GetDetailsResponse{
+		Detail: &pr12er.Detail{
+			Paper: []*pr12er.Paper{
+				{
+					PaperId: "1",
+					Absract: "We propose a new framework for estimating generative models via an adversarial process, in which we simultaneously train two models: a generative model G that captures the data distribution, and a discriminative model D that estimates the probability that a sample came from the training data rather than G. The training procedure for G is to maximize the probability of D making a mistake. This framework corresponds to a minimax two-player game. In the space of arbitrary functions G and D, a unique solution exists, with G recovering the training data distribution and D equal to 1/2 everywhere. In the case where G and D are defined by multilayer perceptrons, the entire system can be trained with backpropagation. There is no need for any Markov chains or unrolled approximate inference networks during either training or generation of samples. Experiments demonstrate the potential of the framework through qualitative and quantitative evaluation of the generated samples.",
```

Comment:
```suggestion
					// nolint: lll 
					Absract: "We propose a new framework for estimating generative models via an adversarial process, in which we simultaneously train two models: a generative model G that captures the data distribution, and a discriminative model D that estimates the probability that a sample came from the training data rather than G. The training procedure for G is to maximize the probability of D making a mistake. This framework corresponds to a minimax two-player game. In the space of arbitrary functions G and D, a unique solution exists, with G recovering the training data distribution and D equal to 1/2 everywhere. In the case where G and D are defined by multilayer perceptrons, the entire system can be trained with backpropagation. There is no need for any Markov chains or unrolled approximate inference networks during either training or generation of samples. Experiments demonstrate the potential of the framework through qualitative and quantitative evaluation of the generated samples.",
```